perception/autoware_map_based_prediction/src/utils.cpp

```diff
@@ -511,7 +511,8 @@ double lateral_distance_to_lanelet_bounds(
   auto distance = std::numeric_limits<double>::max();
   for (const auto & bound : {ll.leftBound(), ll.rightBound()}) {
     const auto p = lanelet::BasicPoint2d(point.x, point.y);
```

Comment:
[IMO] Generating `p` can be moved to the line just before it needs because this might be unused if some conditions are not satisfied, or we can move it to the out of the loop because this is always created from the same `point`.